src/components/actions/Button/Button.tsx

```diff
@@ -34,16 +34,12 @@ export function provideStyles({
 
   return {
     ...DEFAULT_STYLES,
-    ...STYLES_BY_SIZE[size || 'default'],
+    ...STYLES_BY_SIZE[size ?? 'default'],
```

Comment:
Nope, there is no false key, so the first implementation is better.

Reply:
As well as any falsy values like `''` or `0`. I think using proper syntax introduced exactly for these situations is better than using a kinda hacky syntax. That's why I replaced `||` with `??`